setup.py

```diff
@@ -0,0 +1,39 @@
+
+# -*- coding: utf-8 -*-
+
+# DO NOT EDIT THIS FILE!
+# This file has been autogenerated by dephell <3
+# https://github.com/dephell/dephell
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found indentation with spaces instead of tabs](https://app.codacy.com/app/pwwang/pyparam/pullRequest?prid=4632439)